src/iotjs_binding_helper.c

```diff
@@ -145,6 +145,11 @@ int iotjs_process_exitcode() {
   } else {
     exitcode = (uint8_t)iotjs_jval_as_number(num_val);
   }
+
+  uint8_t native_exitcode = iotjs_environment_get()->exitcode;
+  if (native_exitcode != exitcode && native_exitcode) {
+    exitcode = iotjs_environment_get()->exitcode;
```

Comment:
You can use the `native_exitcode` instead of the `iotjs_environment_get()->exitcode`.

Reply:
Right, thanks